addons/mail/static/src/components/attachment_card/attachment_card.xml

```diff
@@ -12,47 +12,47 @@
-                            </div>
+                            </small>
                         </t>
                     </div>
                     <!-- Attachment aside -->
                     <t t-if="(!attachmentCard.attachmentList.composerViewOwner or attachmentCard.attachment.isEditable)">
-                        <div class="o_AttachmentCard_aside position-relative overflow-hidden" t-att-class="{ 'o-has-multiple-action d-flex flex-column': !attachmentCard.attachmentList.composerViewOwner and attachmentCard.attachment.isEditable }">
+                        <div class="o_AttachmentCard_aside position-relative rounded-right overflow-hidden" t-att-class="{ 'o-has-multiple-action d-flex flex-column': !attachmentCard.attachmentList.composerViewOwner and attachmentCard.attachment.isEditable }">
                             <!-- Uploading icon -->
                             <t t-if="attachmentCard.attachment.isUploading and attachmentCard.attachmentList.composerViewOwner">
-                                <div class="o_AttachmentCard_asideItem o_AttachmentCard_asideItemUploading d-flex justify-content-center align-items-center" title="Uploading">
+                                <div class="o_AttachmentCard_asideItem o_AttachmentCard_asideItemUploading d-flex justify-content-center align-items-center w-100 h-100" title="Uploading">
                                     <i class="fa fa-spin fa-spinner"/>
                                 </div>
                             </t>
                             <!-- Uploaded icon -->
                             <t t-if="!attachmentCard.attachment.isUploading and attachmentCard.attachmentList.composerViewOwner">
-                                <div class="o_AttachmentCard_asideItem o_AttachmentCard_asideItemUploaded d-flex justify-content-center align-items-center" title="Uploaded">
+                                <div class="o_AttachmentCard_asideItem o_AttachmentCard_asideItemUploaded d-flex justify-content-center align-items-center w-100 h-100 text-primary" title="Uploaded">
                                     <i class="fa fa-check"/>
                                 </div>
                             </t>
                             <!-- Remove button -->
                             <t t-if="attachmentCard.attachment.isEditable">
-                                <div class="o_AttachmentCard_asideItem o_AttachmentCard_asideItemUnlink d-flex justify-content-center align-items-center" t-att-class="{ 'o-pretty position-absolute': attachmentCard.attachmentList.composerViewOwner }" t-on-click="attachmentCard.onClickUnlink" title="Remove">
+                                <a class="o_AttachmentCard_asideItem o_AttachmentCard_asideItemUnlink top-0 d-flex justify-content-center align-items-center w-100 h-100 bg-300 o_cursor_pointer" t-att-class="{ 'o-pretty position-absolute btn-primary': attachmentCard.attachmentList.composerViewOwner }" t-on-click="attachmentCard.onClickUnlink" title="Remove">
```

Comment:
Hello, thank you for the PR. I've not checked everything else yet, but I don't think these tags should be changed to be links, as they are not links (and they don't even have `href` which is problematic). If you really need to change the tag (why?) then I suppose it should be `button`.

Reply:
Hello 👋 ,
Changing the tag to a link or a button allows to use the `hover` state already defined in the `bg-300` rather than using raw css.
I changed the tag to a `button` 👍 .